apiserver/facades/client/highavailability/package_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/juju/juju/internal/testing"
 )
 
+//go:generate go run go.uber.org/mock/mockgen -typed -package highavailability -destination common_mock_test.go github.com/juju/juju/apiserver/facades/client/highavailability ModelInfoService
```

Comment:
No need for this since the test is skipped.

Reply:
fixed.